stdlib/stdlib.mli

```diff
@@ -998,7 +998,12 @@ val seek_out : out_channel -> int -> unit
 val pos_out : out_channel -> int
 (** Return the current writing position for the given channel.  Does
     not work on channels opened with the [Open_append] flag (returns
-    unspecified results). *)
+    unspecified results).
+    For files opened in text mode under Windows, the returned position
+    is approximate (owing to end-of-line conversion).  Saving the
+    current position with [pos_out], then going back to this position
+    using [seek_out] will not work reliably, unless the file is opened
+    in binary mode. *)
```

Comment:
"unless the file is opened in binary mode" : this is redundant with the assumption of the previous sentence (and if that assumption doesn't apply to this final sentence, one should repeat "under Windows").  Proposal:

> For files opened in text mode under Windows, the returned position
>  is approximate (owing to end-of-line conversion) : in particular, saving the
>  current position with [pos_out], then going back to this position
>  using [seek_out] will not work reliably.

Reply:
I was trying to say that if you want to do this (save the current position, then go back to it later), and be portable to Windows, the file must be opened in binary mode.  Redundancy is good sometimes.

Reply:
Fair enough.  I was concerned that the ocamldoc output for the text will not make the scoping of the assumption clear, and that the partial redundancy would add to the confusion.  Perhaps force a paragraph break before "For files opened..."?

Reply:
I made it clearer in the final version I'm about to merge.